<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>812</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>P1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Pp_1"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>P2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="Pp_2"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>P3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="Pp_3"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>I1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Ip_1"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="Ip_2"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>I3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="Ip_3"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>D1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Dp_1"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>D2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="Dp_2"/>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>D3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="Dp_3"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>I2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>812</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mositure Plant 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="MOS_PLANT1"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Mositure Plant 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="MOS_PLANT2"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Mositure Plant 3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="MOS_PLANT3"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>PWM Plant 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="PWM_PLANT1"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>PWM Plant 2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="PWM_PLANT2"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>PWM Plant 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="PWM_PLANT3"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Oz of Water Today Plant 1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="WATER_PLANT1"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Oz of Water Today Plant 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="WATER_PLANT2"/>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Oz of Water Today Plant 3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLineEdit" name="WATER_PLANT3"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>521</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Comm String </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="COMM_DATA"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Update_Constants">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Constants</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
